### PR TITLE
Feature/image input

### PR DESCRIPTION
**Pull request by litvinets**
Image Input converted into a custom Form Control, added an option to select multiple pictures

### CODE REVIEW

File: src/app/shared/directives/min-max.directive.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  @Input()
+  public max: number;
+
+  constructor(private ref: ElementRef) { }
+
+  @HostListener('input', [ '$event' ])
+    public onInput(a_Event: InputEvent): void {
+      let val = parseInt(this.ref.nativeElement.value);
+      if(this.max !== null && this.max !== undefined  && val >= this.max)
```
Comment (dmitryy90) [inline src/app/shared/directives/min-max.directive.ts:19]: indentations

File: src/app/shared/directives/min-max.directive.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  @Input()
+  public min: number;
+
+  @Input()
+  public max: number;
+
+  constructor(private ref: ElementRef) { }
+
+  @HostListener('input', [ '$event' ])
```
Comment (dmitryy90) [inline src/app/shared/directives/min-max.directive.ts:16]: input event triggers only after focus out, did you try keypress?

File: src/app/shell/provider/provider-activities/create-activity/create-about-form/create-about-form.component.ts [hunk 3]
```diff
@@ -1,62 +1,43 @@
...
       type: new FormControl(''),
-      title: new FormControl(''), 
-      phone: new FormControl(''),
-      email: new FormControl(''),
-      ageFrom: new FormControl(''),
-      ageTo: new FormControl(''),
-      classAmount: new FormControl(''),
+      phone: new FormControl('', [Validators.required, Validators.maxLength(9), Validators.minLength(9)]),
+      email: new FormControl('', [Validators.required, Validators.email]),
+      minAge: new FormControl('', [Validators.required, Validators.maxLength(2)]),
```
Comment (dmitryy90) [inline src/app/shell/provider/provider-activities/create-activity/create-about-form/create-about-form.component.ts:24]: as a suggestion, you can try Validators.max(16), Validators.min(1) also

**Review by dmitryy90 — CHANGES_REQUESTED** [review]

File: src/app/shell/provider/provider-activities/create-activity/create-about-form/create-about-form.component.ts (hunk 3 above)
Comment (litvinets) [reply src/app/shell/provider/provider-activities/create-activity/create-about-form/create-about-form.component.ts:24]: I don't know why, but it didn't work:/ 
should I try more and get rid of the directive?

**Review by dmitryy90 — APPROVED** [review]

**Pull request merged**